components/style/core/base.less

```diff
@@ -61,6 +61,10 @@ a {
   cursor: pointer;
   transition: color .3s ease;
 
+  &:focus {
+    text-decoration: underline;
```

Comment:
I just talked to our designer and they are OK with your underline suggestion, so I gone merging it now and tweak some underline styles later. Thx for you contribution! 😃